demo/api/src/config/config.ts

```diff
@@ -89,8 +89,8 @@ export function createConfig(processEnv: NodeJS.ProcessEnv) {
             sendAllMailsBcc: envVars.MAILER_SEND_ALL_MAILS_BCC,
 
             transport: {
-                host: "localhost",
-                port: 1025,
+                host: envVars.MAILER_HOST,
+                port: envVars.MAILER_PORT,
```

Comment:
please create an extra PR, this is unrelated to mailer logging - you even have already an extra commit